src/components/backup/RestoreCloudStep.js

```diff
@@ -199,7 +199,16 @@ export default function RestoreCloudStep({
           const p1 = dispatch(walletsSetSelected(firstWallet));
           const p2 = dispatch(addressSetSelected(firstAddress));
           await Promise.all([p1, p2]);
-          await initializeWallet(null, null, null, false, false, null, true);
+          await initializeWallet(
+            null,
+            null,
+            false,
+            false,
+            null,
+            null,
+            null,
+            true
+          );
```

Comment:
i don't have context on this, why do we need to change these methods params?

Reply:
two things have changed here:

1. the `color` parameter is used differently. previously it was only used to pass in a `forceColor` to override the wallets address hashed color. i don't believe `forceColor` is actually used anywhere though, pretty sure its obsolete. now, `color` represents the color fetched from the wallet's rainbow web profile, or the address hashed color if no web profile exists.
2. the `emoji` parameter got added. this represents the emoji fetched from the wallet's rainbow web profile, or the address hashed emoji if no web profile exists.